dlext/src/FixDLExt.cpp

```diff
@@ -40,9 +44,21 @@ FixDLExt::FixDLExt(LAMMPS* lmp, int narg, char** arg)
 }
 
 int FixDLExt::setmask() { return FixConst::POST_FORCE; }
-void FixDLExt::post_force(int) { callback(update->ntimestep); }
+void FixDLExt::post_force(int vflag) 
+{
+    // virial setup
+    v_init(vflag);
+
+    // invoke callback
+    callback(update->ntimestep);
+
+    // put the virial from the bias into this fix's member variable virial[6] (see fix.h)
+    setVirial(virial);
```

Comment:
Does `virial` always live on the CPU regardless of whether you're using Kokkos or not?

Reply:
@pabloferz right, `virial` is a member variable of `Fix`, always there regardless of using Kokkos or not.